libnetwork/network_capability.go

```diff
@@ -0,0 +1,11 @@
+package libnetwork
+
+import "github.com/docker/docker/libnetwork/driverapi"
+
+func (n *Network) hasEndpointCapability() bool {
+	// TODO(aker): remove after the next MCR LTS has been released
+	if n.driverCaps == (driverapi.Capability{}) {
+		return n.Type() != "host" && n.Type() != "null"
+	}
```

Comment:
Is this necessary with `Network.driver()` setting up `n.driverCaps`? (I guess it is, but maybe worth saying why in the comment?)